api/src/pcapi/routes/backoffice/accounts/blueprint.py

```diff
@@ -1232,3 +1239,46 @@ def get_public_account_history(
     history = sorted(history, key=lambda item: item.actionDate or datetime.datetime.min, reverse=True)
 
     return history
+
+
+@public_accounts_blueprint.route("/<int:user_id>/gdpr-extract", methods=["POST"])
+@atomic()
+@utils.permission_required(perm_models.Permissions.EXTRACT_PUBLIC_ACCOUNT)
+def create_extract_user_gdpr_data(user_id: int) -> utils.BackofficeResponse:
+    user = (
+        users_models.User.query.filter(
+            users_models.User.id == user_id,
+            sa.or_(
+                users_models.User.roles.contains([users_models.UserRole.BENEFICIARY]),
+                users_models.User.roles.contains([users_models.UserRole.UNDERAGE_BENEFICIARY]),
+            ),
```

Comment:
Pourquoi seulement si le jeune est bénéficiaire ? Je ne vois pas de limitation en ce sens demandée par le ticket et pense que tous les utilisateurs jeune (ayant ou non commencé leur parcours d'inscription) ou grand public ont le droit de demander leurs données personnelles.

Reply:
Pour l'instant l'extraction de données n'est spécifié que pour les beneficiaires (en effet j'ai oublié de le mettre dans le ticket )

Reply:
@Meewan m'a dit de vérifier que seul les bénéficiaires pouvaient bénéficier d'une extract et donc de vérifier que ce soit des bénéficiaires, j'ai mal compris?

Reply:
non c'est bien ça, j'ai juste pas pensé a le mettre dans le ticket.

Reply:
ok ; tu peux donc simplifier : 
```suggestion
            users_models.User.is_beneficiary,
```